conf/systemd.service

```diff
@@ -9,7 +9,7 @@ Group=__APP__
 WorkingDirectory=__FINALPATH__/
 Environment="__YNH_NODE_LOAD_PATH__"
 Environment="NODE_ENV=production"
-ExecStart=__YNH_NPM__ start
+ExecStart="/opt/node_n/n/versions/node/19/bin/pnpm start"
```

Comment:
Any reason for hard coding the path instead of letting corepack handle it?

Reply:
How corepack will handle it from the service ?
If I run the service in the folder, will it use the local pnpm (right now it doesn't find it, is it because corepack enable wasn't used ?)